components/uuf-renderablecreator-hbs/src/test/java/org/wso2/carbon/uuf/handlebars/HbsHelperTest.java

```diff
@@ -346,4 +347,19 @@ public String render(Model model, Lookup lookup, RequestLookup requestLookup, AP
                 "type=\"text/javascript\"></script>\n";
         Assert.assertEquals(pageRenderable.render(null, lookup, requestLookup, createAPI()), expected);
     }
+
+    @Test
+    public void testCSRFTokenHelper() {
+        RequestLookup requestLookup = createRequestLookup();
+        when(requestLookup.getRequest().getCookieValue(COOKIE_CSRFTOKEN)).thenReturn("A45B3DDE4CF00891E7A9F3B752F18F92");
+
+        String output = createRenderable("{{csrfToken}}").render(null, createLookup(), requestLookup, createAPI());
+        String expected = "<input type=\"hidden\" name=\"csrftoken\" id=\"csrftoken\" " +
+                "value=\"A45B3DDE4CF00891E7A9F3B752F18F92\"/>";
+        Assert.assertEquals(output, expected);
+
+        String formoutput = createRenderable("<form>{{csrfToken}}</form>").render(null, createLookup(),
```

Comment:
+1.

Was trying to simulate the {{csrfToken}} within a <form> but I guess it doesn't add value, so removed it.